build.gradle

```diff
@@ -1222,6 +1225,41 @@ project(':metadata') {
   }
 }
 
+project(':group-coordinator') {
+  archivesBaseName = "kafka-group-coordinator"
+
+  dependencies {
+    implementation project(':server-common')
+    implementation project(':clients')
+    compileOnly libs.log4j
+    testImplementation libs.junitJupiter
+    testImplementation libs.jqwik
+    testImplementation libs.hamcrest
+    testImplementation libs.mockitoCore
+    testImplementation libs.mockitoInline
```

Comment:
Do we need all these test dependencies? I'd consider removing `mockitoInline` and `hamcrest`.

Reply:
I removed all the unnecessary ones and I kept the minimal that I need for now.